test/EditorGuidelinesTests/OpenFileTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Paul Harrington.  All Rights Reserved.  Licensed under the MIT License.  See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using EditorGuidelinesTests.Harness;
+using Microsoft.VisualStudio;
+using WindowsInput.Native;
+using Xunit;
+
+namespace EditorGuidelinesTests
+{
+    [VsTestSettings(ReuseInstance = false, Version = "[15.0-)")]
```

Comment:
📝 Everything in this project will work as-is with Visual Studio 2012+. The only reason for this limitation in `_testProjectFile` is written to disk as an SDK project (2017+). If the string literal is changed to use the old project style, this can be changed to `[11.0,-)` and the tests will run in all supported versions provided you have them installed.